src/plugins/Clamp.ts

```diff
@@ -22,45 +22,45 @@ import type { Viewport } from '../Viewport';
 export interface IClampOptions
 {
     /**
-    * Clamp left; true = 0
-    *
-    * @default false
-    */
+   * Clamp left; true = 0
+   *
+   * @default false
+   */
     left?: number | boolean | null;
 
     /**
-    * Clamp top; true = 0
-    *
-    * @default false
-    */
+   * Clamp top; true = 0
+   *
+   * @default false
+   */
     top?: number | boolean | null;
 
     /**
-     * Clamp right; true = viewport.worldWidth
-     *
-     * @default false
-     */
+   * Clamp right; true = viewport.worldWidth
```

Comment:
@davidfig my prettier is not recognizing the eslint, so when I save the files, it gets formated totally differently (all files I save) - so before submitting the PR I run `yarn eslint:fix` to apply the correct linting, and that was the result

Reply:
ok, no problem